### PR TITLE
Update SSHKey to include expires_at as defined by API doc

### PR DESCRIPTION
**Pull request by pschwartz**
Signed-off-by: Philip Marc Schwartz <Philip.Schwartz1@T-Mobile.com>

### CODE REVIEW

**Comment by pschwartz** [comment]
@svanharmelen Another fix if your able to review.

**Review by marceloboeira — APPROVED** [review]

File: users.go [hunk 1]
```diff
@@ -396,6 +396,7 @@ type SSHKey struct {
 	Title     string     `json:"title"`
 	Key       string     `json:"key"`
 	CreatedAt *time.Time `json:"created_at"`
+	ExpiresAt *time.Time `json:"expires_at"`
```
Comment (svanharmelen) [inline users.go:399]: Looking at the docs, should both of these be of type `*ISOTime`?

Comment (pschwartz) [reply users.go:399]: Would be effectively the same. Will change to match other response structs.

**Review by svanharmelen — APPROVED** [review]
Thanks, LGTM 👍🏻

**Pull request merged**